### PR TITLE
LPS-154814

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi Tina, Shuyang,

This is for ticket https://issues.liferay.com/browse/LPS-154814.
Thanks for checking.

### CODE REVIEW

**Comment by catarineaguiar** [comment]
@jeffwu0724 no regressions found.

File: modules/apps/imageio-plugins/imageio-plugins/build.gradle [hunk 1]
```diff
@@ -3,21 +3,21 @@ dependencies {
...
-	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-metadata", version: "3.5"
-	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-pcx", version: "3.5"
-	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-pict", version: "3.5"
-	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-pnm", version: "3.5"
-	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-psd", version: "3.5"
-	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-sgi", version: "3.5"
-	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-tga", version: "3.5"
-	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-thumbsdb", version: "3.5"
-	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-tiff", version: "3.5"
+	compileInclude group: "com.twelvemonkeys.imageio", name: "imageio-bmp", version: "3.8.2"
```
Comment (tinatian) [inline modules/apps/imageio-plugins/imageio-plugins/build.gradle:6]: @jeffwu0724 These three seem related to others, can you also upgrade them?

	compileInclude group: "com.twelvemonkeys.common", name: "common-image", version: "3.3.2"
	compileInclude group: "com.twelvemonkeys.common", name: "common-io", version: "3.3.2"
	compileInclude group: "com.twelvemonkeys.common", name: "common-lang", version: "3.3.2"

**Comment by tinatian** [comment]
@jeffwu0724 Please see my comment and resend.

Thanks.
Tina

**Comment by tinatian** [comment]
ci:close

**Pull request closed without merging**